filter/filter_impl/generic_service_filter.go

```diff
@@ -60,39 +59,53 @@ func (ef *GenericServiceFilter) Invoke(ctx context.Context, invoker protocol.Inv
 	if invocation.MethodName() != constant.GENERIC || len(invocation.Arguments()) != 3 {
 		return invoker.Invoke(ctx, invocation)
 	}
+	logger.Debugf("[Generic Service Filter] arguments: %v", invocation.Arguments())
+	logger.Debugf("[Generic Service Filter] attachments: %v", invocation.Attachments())
 
 	var (
-		ok         bool
-		err        error
-		methodName string
-		newParams  []interface{}
-		genericKey string
-		argsType   []reflect.Type
-		oldParams  []hessian.Object
+		isAssignSuccess bool
```

Comment:
Why don't use ok? It seems that ok and isAssginSuccess are the same stuff.

Reply:
> Why don't use ok? It seems that ok and isAssginSuccess are the same stuff.

updated.